extensions/spark/kyuubi-spark-authz/pom.xml

```diff
@@ -70,17 +72,13 @@
                     <artifactId>logback-classic</artifactId>
                 </exclusion>
                 <exclusion>
-                    <groupId>commons-lang</groupId>
-                    <artifactId>commons-lang</artifactId>
+                    <groupId>org.apache.commons</groupId>
+                    <artifactId>commons-configuration2</artifactId>
```

Comment:
Is this dependency not needed at runtime?

Reply:
seems not, with a basic search on the Ranger codebase, only used from LDAP stuff.